src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import {hot} from "react-hot-loader";
```

Comment:
in official documentation is said `Make sure react-hot-loader is required before react and react-dom:`

Reply:
@NikitaSedyx , Could you help, Is it really vital or not ?

Reply:
I can't argue with documentation :)

it can work now and produce some issues later, who knows

Reply:
fixed